pkgs/by-name/ru/ruff/package.nix

```diff
@@ -22,26 +26,46 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-//ayB5ayYM5FqiSXDDns2tIL+PJ0Osvkp8+MEEL0L+8=";
   };
 
-  cargoLock = {
+  postPatch = ''
+    sed -i -f ${./ruff_exe.sed} python/ruff/__main__.py
+    sed -i 's#@RUFF_BIN@#${placeholder "bin"}/bin/ruff#' python/ruff/__main__.py
```

Comment:
Why not use a `.patch` file ? (and/or `substituteInPlace` ?)

Reply:
The content I'm removing is larger than I'm confident with a patch. That inflates the size of the patch and makes it brittle.

That removes this:
https://github.com/astral-sh/ruff/blob/2f88f849726d50de91efc48ac0148395b7231cd0/python/ruff/__main__.py#L9-L57

Reply:
Wouldn't the following be enough ?
```
substituteInPlace python/ruff/__main__.py \
  --replace-fail 'ruff_exe = "ruff" + sysconfig.get_config_var("EXE")' 'return "${placeholder "$out"}/bin/ruff"'
```
?

Reply:
yeah, that works.